Makefile

```diff
@@ -35,25 +35,38 @@ help:
 list:
 	@echo ${CITIES}
 
+$(filter %,$(CITIES)): %: \
+  ${DATA_DIR}/%.bbox \
+  ${DATA_DIR}/%.osm.pbf \
+  ${DATA_DIR}/%.gtfs.tar \
+  ${DATA_DIR}/%.nominatim.sql ${DATA_DIR}/%.nominatim_tokenizer.tgz \
+  ${DATA_DIR}/%.photon.tgz \
+  ${DATA_DIR}/%.mbtiles \
+  ${DATA_DIR}/%.graph.obj \
+  ${DATA_DIR}/%.valhalla.tar \
+  tag_images
+	@echo "Built $@"
+
+%.bbox:
+	@echo "Extracting bounding box for $(notdir $*)"
```

Comment:
Ideally we could remove `${DATA_DIR}/bbox.txt` completely and just expand the HEADWAY_AREA environment variable in the nginx config to alias to `${HEADWAY_AREA}.bbox` instead of bbox.txt

https://github.com/ellenhp/headway/blob/main/web/nginx.conf.template#L74

I haven't gotten around to figuring that out. The bbox.txt being regenerated every time make is invoked was a bandaid solution to make it easier to recover from an inconsistent state.

Reply:
I took a shot at this in #40